Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -31,10 +31,13 @@
 #import "PlatformCALayerRemote.h"
 #import "RemoteLayerTreeHost.h"
 #import <QuartzCore/QuartzCore.h>
-#import <RealitySystemSupport/RealitySystemSupport.h>
 #import <pal/spi/cocoa/QuartzCoreSPI.h>
 #import <wtf/SoftLinking.h>
 
+#if PLATFORM(VISION)
+#import <RealitySystemSupport/RealitySystemSupport.h>
+#endif
```

Comment:
This isn't enough unfortunately, I'm still hitting:

```
In file included from /Volumes/Data/Code/Safari/OpenSource/WebKitBuild/Debug/DerivedSources/WebKit/unified-sources/UnifiedSource41-mm.mm:7:
/Volumes/Data/Code/Safari/OpenSource/Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm:70:13: error: unknown type name 'RCPRemoteEffectInputTypes'
            RCPRemoteEffectInputTypes allowedInputs = RCPRemoteEffectInputTypesAll ^ RCPRemoteEffectInputTypePointer;
/Volumes/Data/Code/Safari/OpenSource/Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm:70:55: error: use of undeclared identifier 'RCPRemoteEffectInputTypesAll'
            RCPRemoteEffectInputTypes allowedInputs = RCPRemoteEffectInputTypesAll ^ RCPRemoteEffectInputTypePointer;
/Volumes/Data/Code/Safari/OpenSource/Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm:70:86: error: use of undeclared identifier 'RCPRemoteEffectInputTypePointer'
            RCPRemoteEffectInputTypes allowedInputs = RCPRemoteEffectInputTypesAll ^ RCPRemoteEffectInputTypePointer;
3 errors generated.
```

Reply:
Are you sure you haven't changed the value of `ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)` locally? This file is not intended to build on other platforms.